q2_feature_table/tests/test_filter.py

```diff
@@ -203,6 +203,44 @@ def test_sample_metadata(self):
         expected = Table(np.array([]), [], [])
         self.assertEqual(actual, expected)
 
+        # exclude one
```

Comment:
All of the below tests are based on those for filtering (e.g, `exclude one` is based on `filter one`, etc), the difference being that I expect the opposite result since `exclude_ids=True`.